pkg/validation/spec/gnostic.go

```diff
@@ -0,0 +1,1514 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package spec
```

Comment:
sorry, leave a blank line between the comment and the package decl so this doesn't show up in godoc as package docs (same in the other file)

Reply:
fixed and noted for future